### PR TITLE
MAGENTO2-520 - Failed add_order - Braintree getMappedCCType must be o…

### PR DESCRIPTION
**Pull request by awesselburg**
…f type string, null returned

# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I updated the CHANGELOG.md

### CODE REVIEW

File: src/Test/Integration/Data/Payment/Braintree/CreditCard.php [hunk 1]
```diff
@@ -31,9 +31,11 @@ class CreditCard
     /**
      * Outsourced into a function for re-use
      *
+     * @param string|null $cartType
+     *
      * @return array
      */
-    public static function getAdditionalPayment(): array
+    public static function getAdditionalPayment($cartType = null): array
```
Comment (augsteyer) [inline src/Test/Integration/Data/Payment/Braintree/CreditCard.php]: can't be null, can it?

Comment (awesselburg) [reply src/Test/Integration/Data/Payment/Braintree/CreditCard.php]: yes, in case of "Not mapped payment method"

Comment (augsteyer) [reply src/Test/Integration/Data/Payment/Braintree/CreditCard.php]: kk

Comment (augsteyer) [inline src/Test/Integration/Data/Payment/Braintree/CreditCard.php]:
```suggestion
    public static function getAdditionalPayment(string $cartType = null): array
```

**Pull request merged**